apps/block_scout_web/lib/block_scout_web/endpoint.ex

```diff
@@ -71,7 +71,14 @@ defmodule BlockScoutWeb.Endpoint do
   plug(BlockScoutWeb.Prometheus.Exporter)
 
   # 'x-apollo-tracing' header for https://www.graphqlbin.com to work with our GraphQL endpoint
-  plug(CORSPlug, headers: ["x-apollo-tracing" | CORSPlug.defaults()[:headers]])
+  # TODO(mhowell): Use CURRENT_ENV to conditionally add localhost:3000 to CORS policy, just testing now
+  plug(CORSPlug, 
+      headers: ~w(content-type authorization x-requested-with accept origin x-apollo-tracing),
+      origins: ["http://localhost:3000", "https://app.snapchain.dev"],
```

Comment:
nice!!! can we also add "*-snapchain.vercel.app/"? or just *.vercel.app

<img width="570" alt="image" src="https://github.com/Snapchain/sc-hermez-blockscout/assets/111917526/a321777e-f7b4-4316-930a-263841332bc8">

Reply:
https://stackoverflow.com/questions/14003332/access-control-allow-origin-wildcard-subdomains-ports-and-protocols

Reply:
gotcha. can we add https://sc-webapp-bap2pecs-snapchain.vercel.app/ and the one with your handle?

Reply:
kk